### PR TITLE
Api GET request to fetch dummy data from mongoDb

### PR DESCRIPTION
**Pull request by spraguejdev**
@SduongPlanD Can you review my PR? 

To start this application, checkout the readme file that is included in this PR

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -17,15 +18,10 @@
 
 ## Usage
 
-> Some usage instructions
+>  From the root directory of this service, grab dependencies by running npm install. Then run npm start to get the server going on localhost:3000. Then run npm run react-dev to transpose the es6 files into a bundle using webpack. Start your mongodb by typing in mongod into the console. Next, npm run seed-db. This script will create a properties database with 3 tables (properties comparablehomes and localhomes), and then populate the three tables with 100 documents each.
```
Comment (leightonchen) [inline README.md]: Can move the dependency text to the section down below

File: database/db.js [hunk 2]
```diff
@@ -110,7 +110,7 @@ Property.insertMany(dummyData.propertyData, (err, data) => {
 });
 
 // Insert dummydata into comparableHomes model
-ComparableHomes.insertMany(dummyData.comparableHomeData, (err, data) => {
```
Comment (leightonchen) [inline database/db.js:113]: Earlier, when I said that you should have seeding in one file, I meant that you should have a separate script named `seeder.js` or similarly, and have `npm run seed-data` run that instead. The way it's done here, every time the server is run, it re-seeds the collection with fake data. 

My mistake if I was unclear before!

**Review by leightonchen — CHANGES_REQUESTED** [review]
Have two separate files: 
`db.js`: Handles getting/inserting into collection, specifically for server use.
`seeder.js`: Handle only inserting fake data. Don't ever have this be run outside of `npm run seed-data`.

File: README.md (hunk 1 above)
Comment (spraguejdev) [reply README.md]: @leightonchen made this change. Thanks!

File: database/db.js (hunk 2 above)
Comment (spraguejdev) [reply database/db.js:113]: @leightonchen thanks for the tip. I made this change as well!

**Pull request merged**